### DISCUSSION TITLE
how to compile it to get it working ?

### DISCUSSION THREAD

**Discussion by help40** [tjboldt/ProDOS-Utilities#2] [General]
you are using "ProDOS-Utilities" as command ... how to get it compiled and working ?

**Comment by tjboldt** [comment]
Thanks for taking an interest in this project. To get started, you need to get the source code (either download directly from this project or preferably using git to do a git clone). You will then need to install Go from https://go.dev/ and then cd to the source directory where you cloned or downloaded this source and run ‘go build’. This will generate the command line executable for your platform.  Once you have the compiled program, you can follow the instructions in the README.md file included in the source. I will update the README with compilation and getting started instructions.

**Comment by help40** [comment]
I will try to build it with "run ‘go build’" , it is now complaining about missing PATHs , but I will try to "fix" it or to put source in the right path .. , BTW which version does your code need to work

**Comment by tjboldt** [comment]
As per the go.mod file in the project it is expecting Go 1.16. I realize that it would be useful for people just wanting to try out the tool to not have to build from source so I built binaries for most popular platforms and added links in the README at the root of this project. Let me know if that works for you.
